include/cuco/detail/static_map/kernels.cuh

```diff
@@ -108,4 +108,105 @@ __global__ void insert_or_apply(InputIt first, cuco::detail::index_type n, Op op
   }
 }
 
+template <int32_t CGSize,
+          int32_t BlockSize,
+          typename SharedMapRefType,
+          typename InputIt,
+          typename Op,
+          typename Ref>
```

Comment:
nit: Use `class` instead of `typename`. I know we're not consequently enforcing this guideline in cuco, but I would suggest we apply it for every new code snippet that we merge into this library.

Reply:
Thanks! Updated in [3039753](https://github.com/NVIDIA/cuCollections/pull/551/commits/303975392f2b2b77f31ecd299c7df7580153225d)